### ISSUE TITLE
Combine Organizer View and Admin View into one Competition Form

### ISSUE THREAD

**Issue by pedrosino** [thewca/worldcubeassociation.org#1331]
See https://github.com/thewca/worldcubeassociation.org/issues/1069#issuecomment-270747909 and https://github.com/thewca/worldcubeassociation.org/pull/1073

**Comment by dunkOnIT** [comment]
@gregorbg any reason why we wouldn't do this? Seems reasonable to me

**Comment by maxidragon** [comment]
I don't think it's a good idea. For example, when someone has admin permissions and organizes/delegates a comp, they could accidentally execute something they normally wouldn't be able to. If we want to merge these two forms, we should add a switch to activate admin permissions.

**Comment by dunkOnIT** [comment]
Good point - this is probably better implemented by a more nuanced permission system as described in #8237

**Comment by dunkOnIT** [comment]
Wait this is still a good idea it's just blocked by #8237

**Comment by gregorbg** [comment]
> Wait this is still a good idea it's just blocked by #8237

I'm not sure whether this is genuinely blocked. The kind of abuse you're describing in that issue is already possible with our status quo right now. In theory, if someone were to submit a PR for this Issue today, it would not introduce any more new vulnerabilities or attack vectors for abuse.

**Comment by dunkOnIT** [comment]
The problem isn't abuse, the problem is users with elevated permissions accidentally changing settings that WCAT should change - think me/Finn being able to sign up for a comp when we shouldn't be.

Maks's point is that currently, an organizer with admin privileges would still use the Organizer View to make changes to the comp - and thus, wouldn't accidentally override any settings.

#8237 would introduce the ability for users to consciously _activate_ their elevated permissions when needed, meaning that by default a user with access to admin privileges would only be able to act as an organizer unless they "toggle on" their admin privileges - meaning that the kinds of accidental changes we'd be concerned about get much harder to do.

**Comment by gregorbg** [comment]
But the suggestion previously made here was that the Competitions Form UI would get a toggle switch to be able to enable/disable Admin view within the form on purpose.

The issue that you're referencing as blocker would be another high-level safety net _on top of_ such a toggle. I.e. the user could go into their settings to "leave sudo mode" and make sure that the toggle on the Comp Form itself doesn't even appear in the first place.

But that's not strictly blocking the perspective of introducing a local, comp-form-only toggle to switch between organizer and admin privileges within the same page view.

**Comment by maxidragon** [comment]
I agree with Gregor here, we are talking about two different things. Introducing a switch to activate the admin view of the competition form is a simple task that could be done now. Turning off "sudo mode" is also a nice feature, but it's a bit more complex. It can be done later and doesn't block this issue.

**Comment by dunkOnIT** [comment]
Ah cool makes sense! Changing title
